[2021-08-21 01:25:31,703] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-08-21T01:25:26.638183+00:00 [queued]>
[2021-08-21 01:25:31,724] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-08-21T01:25:26.638183+00:00 [queued]>
[2021-08-21 01:25:31,725] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-21 01:25:31,725] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-21 01:25:31,725] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-21 01:25:31,739] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): processing_user> on 2021-08-21T01:25:26.638183+00:00
[2021-08-21 01:25:31,745] {standard_task_runner.py:52} INFO - Started process 11858 to run task
[2021-08-21 01:25:31,748] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'processing_user', '2021-08-21T01:25:26.638183+00:00', '--job-id', '31', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp0qduev06', '--error-file', '/tmp/tmpnrm4pdht']
[2021-08-21 01:25:31,748] {standard_task_runner.py:77} INFO - Job 31: Subtask processing_user
[2021-08-21 01:25:31,803] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2021-08-21T01:25:26.638183+00:00 [running]> on host 07cd8e03eca8
[2021-08-21 01:25:31,876] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-08-21T01:25:26.638183+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-21T01:25:26.638183+00:00
[2021-08-21 01:25:31,894] {python.py:151} INFO - Done. Returned value was: None
[2021-08-21 01:25:31,911] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20210821T012526, start_date=20210821T012531, end_date=20210821T012531
[2021-08-21 01:25:31,951] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-21 01:25:31,959] {local_task_job.py:149} INFO - Task exited with return code 0
